lib/evss/letters/download_configuration.rb

```diff
@@ -2,9 +2,10 @@
 module EVSS
   module Letters
     class DownloadConfiguration < EVSS::Letters::Configuration
+      self.read_timeout = Settings.evss.letters.timeout || 55
+
       def connection
-        @conn ||= Faraday.new(base_path, ssl: ssl_options) do |faraday|
-          faraday.options.timeout = DEFAULT_TIMEOUT
+        @conn ||= Faraday.new(base_path, request: request_options, ssl: ssl_options) do |faraday|
```

Comment:
I don't see `request_options` defined anywhere anymore.

Reply:
@omgitsbillryan request_options is defined in the base config class
https://github.com/department-of-veterans-affairs/vets-api/blob/0b4bddd5857850a0a9bf241f6a3cf5850795e90d/lib/common/client/configuration/base.rb#L32-L37